.code-samples.meilisearch.yaml

```
@@ -327,25 +324,25 @@ getting_started_search_md: |-
 
   [About this SDK](https://github.com/meilisearch/meilisearch-js/)
 get_attributes_for_faceting_1: |-
-  client.index('movies').getAttributesForFaceting()
+  client.index('movies').getFilterableAttributes()
 update_attributes_for_faceting_1: |-
   client.index('movies')
-    .updateAttributesForFaceting([
+    .updateFilterableAttributes([
       'genres',
       'director'
     ])
 reset_attributes_for_faceting_1: |-
```

```suggestion
reset_attributes_for_faceting_1: |-
```
should be changed with `reset_filterable_attributes_1` or depending on the name of the section